### ISSUE TITLE
Error Generating Languages.

### ISSUE THREAD

**Issue by DavaGordon** [magento/magento2#36405]
### Preconditions and environment

- Magento version: 2.4.2 p1
- While trying to generate languages files i stumbled across this issue which prevented me from being able complete the task. 

### Steps to reproduce

1. From the CLI 
2. enter `bin/magento i18n:collect-phrases --o="/full/path/to/magento/app/design/frontend/custom/theme/i18n/en_US.csv" --magento ` and hit return

### Expected result

Collection and generation of translatable words in `en_US.csv` generated to directory `/full/path/to/magento/app/design/frontend/custom/theme/i18n/`

### Actual result

In Phrase.php line 90: Missed phrase  
                 

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [X] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by DavaGordon** [comment]
@magento I am working on this

**Comment by DavaGordon** [comment]
@magento add to contributors team

**Comment by DavaGordon** [comment]
@magento add to contributors team

**Comment by DavaGordon** [comment]
https://github.com/magento I am working on this

**Comment by DavaGordon** [comment]
@magento I am working on this

**Comment by engcom-November** [comment]
Hi @DavaGordon , Thanks for the report and collaboration!
We have tried to reproduce this issue in Magento 2.4-develop branch but the issue is not reproducible.

We have followed the below steps:
1. Ran the below command from cli: 
bin/magento i18n:collect-phrases --o="/full/path/to/magento/app/design/frontend/custom/theme/i18n/en_US.csv" --magento  

Note: Followed this reference to reproduce this issue 
https://aureatelabs.com/blog/how-to-create-and-use-language-package-component-in-magento-2/  

Do we miss anything?
Thank you.
